.github/workflows/test_branches.yml

```diff
@@ -259,6 +259,13 @@ jobs:
         # Note: some pypi packages are not available through conda
         PYOMO_DEPENDENCIES=`python setup.py dependencies \
             --extras "$EXTRAS" | tail -1`
+        # HACK: Remove problem packages on conda+Linux
+        if test "${{matrix.TARGET}}" == linux; then
+            WORDSTOREMOVE="casadi numdifftools pint"
+            for WORD in $WORDSTOREMOVE; do
+                PYOMO_DEPENDENCIES=${PYOMO_DEPENDENCIES//$WORD/}
+            done
+        fi
```

Comment:
Are the changes to GitHub Actions still necessary? Do you need to merge main into this branch?

Reply:
Sorry. I forgot to merge main into this branch. Fixed.